osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableHit.cs

```diff
@@ -51,6 +53,9 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
 
         public override bool OnPressed(TaikoAction action)
         {
+            if (handleExtraPress)
```

Comment:
i don't like this variable name... because it's doing the opposite of handling the extra press (it's *ignoring* it).


What would you say to making this an `int judgedPressesThisFrame` or `pressHandledThisFrame`